frontend/src/actions/story.js

```diff
@@ -124,12 +124,16 @@ export const getStorySuggestions = (story) => {
   if (
     story.sick === sicknessStatus.RECOVERED ||
     story.sick === sicknessStatus.SICK
-  )
+  ) {
+    let checkSymptomsLink = "https://landing.google.com/screener/covid19";
+    if (story.countryOfOrigin === "United States of America")
```

Comment:
I will approve it, but can we please put the string `"United States of America"` on a constant? With this one are already four occurrences
Thanks 😬

Reply:
It would be beautiful to have an `Countries` Enum, so we could put Mexico there to
Also, it would make it easy for the future know which countries are being considered for specials links

Reply:
Yeah, what I'm looking forward to in the future is maybe having a static countries endpoint on our end based off the countries that we have data for, as to reduce border errors when matching, or something of the sort... but... you know... anyone can dream